model/distro/db.go

```diff
@@ -37,6 +38,31 @@ func FindOne(query db.Q) (Distro, error) {
 	return d, db.FindOneQ(Collection, query, &d)
 }
 
+func FindAllNames() ([]string, error) {
+	out := struct {
```

Comment:
Is it possible to not have this array and have the aggregation unmarshal into a []string? Maybe with a $objectToArray in the last stage?